kernel/aster-nix/src/net/socket/ip/datagram/mod.rs

```diff
@@ -282,7 +280,7 @@ impl Socket for DatagramSocket {
     fn addr(&self) -> Result<SocketAddr> {
         let inner = self.inner.read();
         match inner.as_ref() {
-            Inner::Unbound(unbound_datagram) => Ok(UNSPECIFIED_LOCAL_ENDPOINT.into()),
+            Inner::Unbound(_unbound_datagram) => Ok(UNSPECIFIED_LOCAL_ENDPOINT.into()),
```

Comment:
Instead of `Inner::Unbound(_unbound_datagram)`, using `Inner::Unbound(..)` or `Inner::Unbound(_)` may be more appropriate. Naming an unused local variable could be unnecessary. This suggestion also applies to other `match` or `if let` statements.

Reply:
Yes, the unnecessary naming is a bit ugly. I've changed it accordingly.